mixbench-cuda/lcutil.h

```diff
@@ -67,15 +67,18 @@ static inline bool IsFP16Supported(void){
 static inline void GetDevicePeakInfo(double *aGIPS, double *aGBPS, cudaDeviceProp *aDeviceProp = NULL){
 	cudaDeviceProp deviceProp;
 	int current_device;
+	int clockRate, memoryClockRate;
 	if( aDeviceProp )
 		deviceProp = *aDeviceProp;
 	else{
 		CUDA_SAFE_CALL( cudaGetDevice(&current_device) );
 		CUDA_SAFE_CALL( cudaGetDeviceProperties(&deviceProp, current_device) );
+		CUDA_SAFE_CALL( cudaDeviceGetAttribute(&clockRate, cudaDevAttrClockRate, current_device) );
+		CUDA_SAFE_CALL( cudaDeviceGetAttribute(&memoryClockRate, cudaDevAttrMemoryClockRate, current_device) );
```

Comment:
Can you check if we don't need device properties structure now at all? Perhaps we could use of `cudaDevAttrGlobalMemoryBusWidth` device attribute in place of `deviceProp.memoryBusWidth`. If yes, then we can drop `cudaDeviceProp *aDeviceProp = NULL` function argument.